test/parallel/test-http-client-check-http-token.js

```diff
@@ -20,7 +20,11 @@ server.listen(0, common.mustCall(() => {
   expectedFails.forEach((method) => {
     assert.throws(() => {
       http.request({ method, path: '/' }, common.mustNotCall());
-    }, /^TypeError: Method must be a string$/);
+    }, common.expectsError({
+      code: 'ERR_INVALID_ARG_TYPE',
+      type: TypeError,
+      message: /The "method" argument must be of type string/
```

Comment:
Unless I am missing something, it does not appear to be necessary to use a regular expression here, a string literal should work just fine. By the way, if you want to match an exact string using regular expressions, use start (`^`) and end markers (`$`). Otherwise, the match will succeed even if there is content before or after the expression:

```js
const expr = /The "method" argument must be of type string/;
const str = 'The "method" argument must be of type string. Also, this sentence should not be here';
expr.test(str) === true
```